.github/workflows/continuous-integration-weekly.ym

```diff
@@ -0,0 +1,54 @@
+name: FAST Workflows - Cross Platforms
+
+on:
+  schedule:
+    - cron: '0 0 * * 2'
+    branches:
+    - master
+
+jobs:
+  build_windows:
+    name: Validate on Windows
+
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install Lerna
+      run: npm i -g lerna

```

Comment:
Can we use yarn instead of this npm run (https://classic.yarnpkg.com/en/docs/cli/global/#toc-yarn-global)? 
```
yarn global add lerna
```

Reply:
I see that is what we're using below

Reply:
I chose to use NPM because ADO machines were failing to run on that code.